### PR TITLE
Clarified scopes and the "global" scope

### PR DESCRIPTION
**Pull request by davkean**
Attempted to clarify the difference between the IVsSolution scope and the VS default container. The terms were used interchangeably which was confusing.

### CODE REVIEW

**Comment by davkean** [comment]
I think to make this document a little less confusing, you should just use "scope" everywhere, instead of using "scope" and "context" interchangeably. Thoughts?

**Comment by davkean** [comment]
+ @AArnott, wouldn't mind you looking over this - to see if I've overstepped here. :)

File: doc/overview/scopes.md [hunk 1]
```diff
@@ -3,10 +3,12 @@ Scopes
...
 ### Introduction to scopes
 
-In Visual Studio there are three "scopes" of context related to project
-systems:
+A "scope" is a collection of MEF parts that are tied to the lifecycle of a specific object, or "context". When that object or context is torn down, so are the MEF parts that live in that scope. Scopes themselves can be nested so that parts that live in child scopes can "inherit" or import exports from parent scopes, but not vice versa.
 
-- `IVsSolution`, or the global context. There is exactly one of these in the 
+In Visual Studio, there is one global default "scope" of context and three related to project systems:
+
+- __VS Default Container__. This is the default global scope, where parts in Visual Studio live by default. There is exactly on of these in the process.
```
Comment (AArnott) [inline doc/overview/scopes.md]: typo: exactly *one* of these

File: doc/overview/scopes.md [hunk 2]
```diff
@@ -3,10 +3,12 @@ Scopes
...
+- `IVsSolution`, or the solution context. There is exactly one of these in the
```
Comment (AArnott) [inline doc/overview/scopes.md]: I think we should avoid referring to the `ProjectService` scope as the solution or `IVsSolution` scope. `ProjectService` has the same lifetime as the VS default container, whereas association with a solution suggests it has solution data and its lifetime gets recycled between loaded solutions, which is not correct.

**Pull request merged**